### PR TITLE
[luci-interpreter] Add support for prelu op

### PR DESCRIPTION
**Pull request by ai-moiseev**
Add support for prelu activation op in luci-interpreter.

ONE-DCO-1.0-Signed-off-by: Alexander Moiseev <a.moiseev@samsung.com>

### CODE REVIEW

**Comment by seanshpark** [comment]
Please rebase to apply fix #4044

File: compiler/luci-interpreter/src/kernels/Prelu.cpp [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+{
+
+Prelu::Prelu(const Tensor *input, const Tensor *alpha, Tensor *output)
+    : Kernel({input, alpha}, {output})
+{
+}
+
+void Prelu::configure()
+{
+  if (input()->element_type() != output()->element_type())
```
Comment (s-barannikov) [inline compiler/luci-interpreter/src/kernels/Prelu.cpp:37]: `LUCI_INTERPRETER_CHECK` was introduced recently for such checks.

File: compiler/luci-interpreter/src/kernels/Prelu.cpp [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  }
+}
+
+void Prelu::evalFloat() const
+{
+  auto input_data = getTensorData<float>(input());
+  auto alpha_data = getTensorData<float>(alpha());
+  auto output_data = getTensorData<float>(output());
+  auto size = getTensorShape(input()).FlatSize();
+  for (auto i = input_data, a = alpha_data; i != input_data + size; ++i, ++a)
```
Comment (s-barannikov) [inline compiler/luci-interpreter/src/kernels/Prelu.cpp:77]: It's difficult to understand what's going on. Espesially it is not obvious that all these variables are pointers.
I'd rewrite it to a classical indexed loop and change the types of pointers to `const auto *` / `auto *`.

File: compiler/luci-interpreter/src/kernels/Prelu.h [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef LUCI_INTERPRETER_KERNELS_PRELU_H
+#define LUCI_INTERPRETER_KERNELS_PRELU_H
+
+#include "core/Kernel.h"
+#include "core/KernelParams.h"
```
Comment (s-barannikov) [inline compiler/luci-interpreter/src/kernels/Prelu.h:21]: Seems unused.

Comment (ai-moiseev) [reply compiler/luci-interpreter/src/kernels/Prelu.h]: Done, cleaned up.

File: compiler/luci-interpreter/src/kernels/Prelu.cpp (hunk 1 above)
Comment (ai-moiseev) [reply compiler/luci-interpreter/src/kernels/Prelu.cpp]: Fixed, replaced with macro-wrapped checks

File: compiler/luci-interpreter/src/kernels/Prelu.cpp (hunk 2 above)
Comment (ai-moiseev) [reply compiler/luci-interpreter/src/kernels/Prelu.cpp]: Done, const pointers and indexed loop added

**Review by s-barannikov — APPROVED** [review]
LGTM

**Review by kvochko — APPROVED** [review]

**Review by jinevening — APPROVED** [review]
LGTM

**Pull request merged**